### ISSUE TITLE
Bluray episodes are not detected correctly

### ISSUE THREAD

**Issue by 020794** [jellyfin/jellyfin#9105]
### Please describe your bug

The directory tree is like

/media
>/tv.shows
>>/Blue.Planet.2017
>>> /Blue.Planet.II.2017.S01E01-E03
>>>> /BDMV
>>>>/CERTIFICATE

>>>/Blue.Planet.II.2017.S01E04-E06
>>>>/BDMV
>>>>/CERTIFICATE

/media/tv.shows is added to Jellyfin as a folder of a tv shows library.

Jellfyin can't treat the bluray folders as a whole, but detect every M2TS files in BDMV/STREAM subfolders and marked them as SPECIAL of the show.

### Jellyfin Version

Other

### if other:

10.8.6

### Environment

```markdown
- OS:ubuntu.22.04
- Virtualization:docker
- Clients:browser
- Browser:chrome.108
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nwgat** [comment]
similar issue with BD ISO files, it can detect them but only plays random playlist, would been nice if there was a choose playlist option that lists all the playlists on each BD folder/iso, similar to how jellyfin treats mkv files with different audio/subtitles tracks

**Comment by ukanuk** [comment]
> /media
> 
> > /tv.shows
> > > /Blue.Planet.2017
> > > > /Blue.Planet.II.2017.S01E01-E03
> > > > > /BDMV
> > > > > /CERTIFICATE
> 
> > > > /Blue.Planet.II.2017.S01E04-E06
> > > > > /BDMV
> > > > > /CERTIFICATE

Is that the expected/supported foder structure for when it arrives in 10.9 per fix in #9254? Documentation does not make it clear how this should be done in https://jellyfin.org/docs/general/server/media/shows 

Currently I have all my stuff named liked like this -- in particular, it's unclear to me how to handle multiple episodes per disc.

> /Media
> 
> > /Shows
> > > /Sailor Moon (1992)
> > > > /Sailor Moon (1992) S01-disc1
> > > > > /BDMV
> > > > > /CERTIFICATE
> > > > > /MAKEMKV
> 
> > > > /Sailor Moon (1992) S01-disc2
> > > > > /BDMV
> > > > > /CERTIFICATE
> > > > > /MAKEMKV

**Comment by Shadowghost** [comment]
The first file structure should allow proper metadata fetching in 10.9
